packages/data-point/src/tracing/Span.test.js

```diff
@@ -0,0 +1,121 @@
+const { toNanoSeconds, getHighResolutionTime, Span } = require("./Span");
+
+const handlers = {
+  start: jest.fn()
+};
+
+const tracer = {
+  handlers
+};
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+const mockHrtimeResult = [75238, 684529285];
+const mockHighResolutionTime = 75238684529285;
```

Comment:
is there any reason for using these specific numbers, or are they random?

Reply:
none at all, they were calculated at the moment of writing the test - so not random, but accurate